src/repositories/books.repository.js

```diff
@@ -0,0 +1,88 @@
+const db = require("../../db");
+
+const getAllBooks = async () => {
+  const sqlQuery = `SELECT * FROM books`;
+
+  try {
+    const result = await db.query(sqlQuery);
+    return result.rows;
+  } catch (error) {
+    console.error(error);
+    throw new Error("Database Error");
+  }
+};
+
+const getBookById = async (bookId) => {
+  const sqlQuery = `select * from books where id = $1`;
+  console.log(bookId);
```

Comment:
Always a good idea to keep your logs as clean as possible, so make sure you don't commit unnecessary `console.logs`